lib/Error/frame.js

```diff
@@ -0,0 +1,47 @@
+const chalk = require('chalk')
```

Comment:
Make compatible with `node >= 0.10` 😞 Or we finally drop the EOL node versions 🙃 ?

Reply:
Try use  [v1.1.3](https://github.com/chalk/chalk/blob/v1.1.3/package.json)

Reply:
Ohh sry this was my bad 😛. The is comment is not about `chalk` in particular, but the whole file in general :)